pages/mentor.tsx

```diff
@@ -0,0 +1,121 @@
+import "twin.macro";
+import Image from "next/image";
+import {
+    Box,
+    Flex,
+    VStack,
+    HStack,
+    Grid,
+    Heading,
+    Text,
+    Badge,
+    AspectRatio,
+} from "@chakra-ui/react";
+import { times } from "lodash";
+
+type TabButtonProps = {
+    isActive?: boolean;
+};
+
+function TabButton({ isActive }: TabButtonProps) {
+    return (
+        <Flex
+            tw="border rounded-full w-3 h-3 items-center justify-center"
+            borderColor={isActive ? "#DC6450" : "gray.400"}
+            color={isActive ? "#DC6450" : "gray.400"}>
+            <Box bgColor="currentcolor" rounded="full" w={2} h={2}></Box>
+        </Flex>
+    );
+}
+
+type AttributesProps = {
+    title: string;
+    badges: string[];
+};
+
+function Attributes({ title, badges }: AttributesProps) {
+    return (
+        <VStack align="start">
+            <Text tw="uppercase text-xs font-bold">{title}</Text>
+            <HStack>
+                {badges.map((text) => (
+                    <Badge key={text} tw="rounded-md px-2 py-1 font-medium" color="gray.500">
+                        {text}
+                    </Badge>
+                ))}
+            </HStack>
+        </VStack>
+    );
+}
+
+function ImageDisplay() {
+    return (
+        <VStack w="full" align="start" spacing={4}>
+            <AspectRatio tw="border relative w-full" ratio={1} bgColor="gray.100">
+                {/* sizes=100% is needed here to make next/image work properly */}
+                <Image src="/svg/avatar.svg" sizes="100%" layout="fill" objectFit="contain" />
+            </AspectRatio>
```

Comment:
AspectRatio makes it really easy to constrain content to a certain ratio. There is no easy work to do this with CSS. This saves so much time without having to experiment